website/pages/docs/index.mdx

```diff
@@ -9,17 +9,17 @@ import Features from "../../components/Features";
 <Features page="docs" detailed={false} />
 
 CloudQuery is an [open-source](https://github.com/cloudquery/cloudquery),
-high-performance data integration platform for security and infrastructure teams.
+high-performance data integration framework built for developers.
 
 CloudQuery extracts, transforms, and loads configuration from cloud APIs to
 variety of supported destinations such as databases, data lakes, or streaming platforms
 for further analysis.
 
 ## Why CloudQuery?
 
-- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with the CloudQuery SDK.
-- **Blazing fast**: CloudQuery's concurrency system leverages the excellent Go concurrency model with light-weight Goroutines.
-- **Database agnostic**: CloudQuery can store your configuration in any supported destination such as a database, data lake, or streaming platform for further analysis.
-- **Pre-built queries**: CloudQuery maintains a rich set of SQL queries and Grafana dashboards for asset inventory, CSPMs, Security & Compliance, and Cost use-cases.
-- **Raw access to data**: Decouple data ingestion and have raw access to your data, with which you can build your own security stack and re-use
-  best-of-breed tools for querying (SQL,…), transformation (dbt,…), and visualization (Grafana, Preset, Metabase, PowerBI, …).
+- **Open source**: Easily extendable plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
```

Comment:
```suggestion
- **Open source**: Extensible plugin architecture. Contribute to our official plugins or develop your own with CloudQuery SDK.
```

On the home page we used "extensible". Similar meanings, but let's keep it consistent